Changelog

```diff
@@ -1,209 +1,218 @@
-1.17.0 (June 12th, 2024)

```

Comment:
Too much change in this file... Maybe due to encoding changes.

Reply:
Just encoding from windows to unix

Reply:
Ok. NTC